assets/sass/components/_pagination.scss

```diff
@@ -0,0 +1,25 @@
+nav.row.siblings {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	justify-content: space-between;
+	margin: 0 5px;
+
+	a.nav-flex-left {
```

Comment:
Since `a.nav-flex-left` and `a.nav-flex-right` have identical rules now, they can be combined with the comma `a.nav-flex-left, a.nav-flex-right { ... }`